css/style.css

```diff
@@ -464,10 +496,270 @@ div .taggs {
   color: #fff;
 }
 
-.shape {
-  margin-left: 30%;
-  background: #0d151e;
-  border-radius: 10px;
-  width: 134px;
-  height: 5px;
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    font-family: 'Poppins', sans-serif;
+    background: #e5e5e5;
+  }
+
+  #main-page {
+    left: 0;
+    height: 800px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background: #fff;
+    background-image: url(/images/desktop_main/Header.svg);
+    background-size: 97%;
+    background-repeat: no-repeat;
+    background-position: 20px 80px;
+    margin-bottom: 4rem;
+    padding: 10px 24px;
+  }
+
+  #navbar {
+    overflow: hidden;
+    background-color: #fff;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    margin-left: 8px;
+  }
+
+  .my-logo {
+    margin: 16px 0 0 100px;
+  }
+
+  .navbar-menu {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    gap: 1rem;
+    margin-right: 10%;
+  }
+
+  .navbar-menu a {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .headline {
+    padding: 350px 260px 255px;
+    margin-top: 20px;
+  }
```

Comment:
- [ ] This headline will get squeezed too much at `770px` because the padding is `260px` at both sides at all sizes from 768px above. I think `260px` is good at wider desktop screens but at lower than 1024px for example, you can give it another padding in another media query. that will look good on your design.
your design
![hgf](https://user-images.githubusercontent.com/96683843/213526685-6c3e9ece-b462-4d7b-8645-c67ec94377a8.jpg)